.github/workflows/deploy-services-sandbox.yml

```
@@ -0,0 +1,84 @@
+name: Deploy Services Sandbox Stack to AWS
+
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - hyp3-services-sandbox
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - environment: hyp3-services-sandbox
+            domain: hyp3-services-sandbox.asf.alaska.edu
+            template_bucket: services-hyp3-artifacts
+            image_tag: test
+            product_lifetime_in_days: 14
+            default_credits_per_user: 0
+            default_application_status: APPROVED
+            cost_profile: DEFAULT
+            job_files: >-
+              job_spec/AUTORIFT.yml
+              job_spec/INSAR_ISCE_BURST.yml
+            instance_types: r6id.xlarge,r6id.2xlarge,r6id.4xlarge,r6id.8xlarge,r6idn.xlarge,r6idn.2xlarge,r6idn.4xlarge,r6idn.8xlarge
+            default_max_vcpus: 640
+            expanded_max_vcpus: 640
+            required_surplus: 0
+            security_environment: ASF
+            ami_id: /aws/service/ecs/optimized-ami/amazon-linux-2023/recommended/image_id
+            distribution_url: ''
+
+    environment:
+      name: ${{ matrix.environment }}
+      url: https://${{ matrix.domain }}
+
+    steps:
+      - uses: actions/checkout@v4.2.2
+
+      - uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.V2_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.V2_AWS_SECRET_ACCESS_KEY }}
+          aws-session-token: ${{ secrets.V2_AWS_SESSION_TOKEN }}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.13
+
+      - run: |
+          jq -n '{"Parameters": $ARGS.named}' \
+            --arg VpcId '${{ secrets.VPC_ID }}' \
+            --arg SubnetIds '${{ secrets.SUBNET_IDS }}' \
+            --arg SecretArn '${{ secrets.SECRET_ARN }}' \
+            --arg ImageTag '${{ matrix.IMAGE_TAG }}' \
+            --arg ProductLifetimeInDays '${{ matrix.product_lifetime_in_days }} ' \
+            --arg AuthPublicKey '${{ secrets.AUTH_PUBLIC_KEY }}' \
+            --arg DefaultCreditsPerUser '${{ matrix.default_credits_per_user }}' \
+            --arg DefaultApplicationStatus '${{ matrix.default_application_status }}' \
+            --arg AmiId '${{ matrix.ami_id }}' \
+            --arg DefaultMaxvCpus '${{ matrix.default_max_vcpus }}' \
+            --arg ExpandedMaxvCpus '${{ matrix.expanded_max_vcpus }}' \
+            --arg MonthlyBudget '${{ secrets.MONTHLY_BUDGET }}' \
+            --arg RequiredSurplus '${{ matrix.required_surplus }}' \
+            --arg InstanceTypes '${{ matrix.instance_types }}' \
+            --arg Domain '${{ matrix.domain }}' \
```

Fixing a typo I introduced in the v9.5.0 release:
```suggestion
            --arg DomainName '${{ matrix.domain }}' \
```